### ISSUE TITLE
Suggestion: Opening file does not need to open new inky window

### ISSUE THREAD

**Issue by mehalon** [inkle/inky#97]
Hi,
whenever I open inky and then open an ink file, it opens a new inky window (while a blank/untitled one remains in the background). Is there any reason for that? Otherwise it could directly open the content of an ink file in the very first window.
It might even make sense to let inky remember which file was opened last and suggest to re-open it when inky is started.

**Comment by joethephish** [comment]
Good point! Also, if you're opening a file that's an include of a project that's currently open, it could simply switch to it.

**Comment by dmcondolora** [comment]
Was going to open a similar issue, but thought I'd just bump this one. It would be great if opening Inky made it reopen the last-used Ink file.
